components/omega/configs/Default.yml

```diff
@@ -0,0 +1,27 @@
+omega:
+   TimeManagement:
```

Comment:
Was using lowercase for the root group and uppercase everywhere else a deliberate choice ?

Reply:
This is how it's stylized in the Config UserGuide and the test file created by the Config unit test. Though, I do think it would look better and would be more consistent to capitalize "Omega"